diff --git a/dist/changesets-apply-release-plan.cjs.js b/dist/changesets-apply-release-plan.cjs.js
index a5c67efc0fbaf74981ab495652ea07220e9fcd7c..79f39e63274a6be707b3e442e1b05e805e9f6f4b 100644
--- a/dist/changesets-apply-release-plan.cjs.js
+++ b/dist/changesets-apply-release-plan.cjs.js
@@ -504,12 +504,14 @@ async function prependFile(filePath, data, name, prettierInstance) {
   await writeFormattedMarkdownFile(filePath, newChangelog, prettierInstance);
 }
 async function writeFormattedMarkdownFile(filePath, content, prettierInstance) {
-  await fs__default["default"].writeFile(filePath, prettierInstance ?
-  // Prettier v3 returns a promise
-  await prettierInstance.format(content, _objectSpread2(_objectSpread2({}, await prettierInstance.resolveConfig(filePath)), {}, {
-    filepath: filePath,
-    parser: "markdown"
-  })) : content);
+  if (!prettierInstance) {
+    await fs__default["default"].writeFile(filePath, content);
+    return;
+  }
+
+  const { format } = await import("oxfmt");
+  const result = await format(filePath, content);
+  await fs__default["default"].writeFile(filePath, result.code);
 }
 
 exports["default"] = applyReleasePlan;
diff --git a/dist/changesets-apply-release-plan.esm.js b/dist/changesets-apply-release-plan.esm.js
index b363b0ca1a0743466485f7978cf3bba336cb51a1..0e6f6c89a89dc68fdc4c741f187cc18d8448fb2b 100644
--- a/dist/changesets-apply-release-plan.esm.js
+++ b/dist/changesets-apply-release-plan.esm.js
@@ -460,12 +460,14 @@ async function prependFile(filePath, data, name, prettierInstance) {
   await writeFormattedMarkdownFile(filePath, newChangelog, prettierInstance);
 }
 async function writeFormattedMarkdownFile(filePath, content, prettierInstance) {
-  await fs.writeFile(filePath, prettierInstance ?
-  // Prettier v3 returns a promise
-  await prettierInstance.format(content, _objectSpread2(_objectSpread2({}, await prettierInstance.resolveConfig(filePath)), {}, {
-    filepath: filePath,
-    parser: "markdown"
-  })) : content);
+  if (!prettierInstance) {
+    await fs.writeFile(filePath, content);
+    return;
+  }
+
+  const { format } = await import("oxfmt");
+  const result = await format(filePath, content);
+  await fs.writeFile(filePath, result.code);
 }
 
 export { applyReleasePlan as default };
