diff --git a/dist/changesets-config.cjs.js b/dist/changesets-config.cjs.js
index 996b3c039ead61f2c0f898b72bb0e40c45714133..c84de39fa80704627f1b285e3a54aece43069f74 100644
--- a/dist/changesets-config.cjs.js
+++ b/dist/changesets-config.cjs.js
@@ -263,7 +263,7 @@ let parse = (json, packages) => {
   // Note: assemble-release-plan uses a graph WITH devDeps because it needs to
   // update devDep ranges in package.json even though they don't cause version bumps.
   const ignore = isArray(json.ignore) ? json.ignore : [];
-  if (ignore.length || !privatePackages.version) {
+  if (!privatePackages.version) {
     const dependentsGraph = getDependentsGraph.getDependentsGraph(packages, {
       ignoreDevDependencies: true,
       bumpVersionsWithWorkspaceProtocolOnly: json.bumpVersionsWithWorkspaceProtocolOnly
@@ -271,7 +271,7 @@ let parse = (json, packages) => {
     const packagesByName = new Map(packages.packages.map(x => [x.packageJson.name, x]));
     for (const pkg of packages.packages) {
       if (!shouldSkipPackage.shouldSkipPackage(pkg, {
-        ignore,
+        ignore: [],
         allowPrivatePackages: privatePackages.version
       })) {
         continue;
@@ -284,7 +284,7 @@ let parse = (json, packages) => {
           continue;
         }
         if (shouldSkipPackage.shouldSkipPackage(dependentPkg, {
-          ignore,
+          ignore: [],
           allowPrivatePackages: privatePackages.version
         })) {
           continue;
diff --git a/dist/changesets-config.esm.js b/dist/changesets-config.esm.js
index a9554b0fa5bab2175d7017943289af4b9415d4fa..3be047d63e8e040cb50a7df432c69a26a37845ed 100644
--- a/dist/changesets-config.esm.js
+++ b/dist/changesets-config.esm.js
@@ -235,7 +235,7 @@ let parse = (json, packages) => {
   // Note: assemble-release-plan uses a graph WITH devDeps because it needs to
   // update devDep ranges in package.json even though they don't cause version bumps.
   const ignore = isArray(json.ignore) ? json.ignore : [];
-  if (ignore.length || !privatePackages.version) {
+  if (!privatePackages.version) {
     const dependentsGraph = getDependentsGraph(packages, {
       ignoreDevDependencies: true,
       bumpVersionsWithWorkspaceProtocolOnly: json.bumpVersionsWithWorkspaceProtocolOnly
@@ -243,7 +243,7 @@ let parse = (json, packages) => {
     const packagesByName = new Map(packages.packages.map(x => [x.packageJson.name, x]));
     for (const pkg of packages.packages) {
       if (!shouldSkipPackage(pkg, {
-        ignore,
+        ignore: [],
         allowPrivatePackages: privatePackages.version
       })) {
         continue;
@@ -256,7 +256,7 @@ let parse = (json, packages) => {
           continue;
         }
         if (shouldSkipPackage(dependentPkg, {
-          ignore,
+          ignore: [],
           allowPrivatePackages: privatePackages.version
         })) {
           continue;
