diff --git a/_lib/app/utils/createFetchRequest.js b/_lib/app/utils/createFetchRequest.js
index ff9d03e4d1164993d4becc914d68b2f1279179c3..78810835b61bad1cf02e0498a187a0278729f8eb 100644
--- a/_lib/app/utils/createFetchRequest.js
+++ b/_lib/app/utils/createFetchRequest.js
@@ -5,6 +5,7 @@ export function createFetchRequest(req) {
     req.on('close', () => controller.abort());
     const headers = new Headers();
     for (const [key, values] of Object.entries(req.headers)) {
+        if (key.startsWith(':')) continue;
         if (values) {
             if (Array.isArray(values))
                 for (const value of values)
